deploy/charts/rook-ceph-cluster/values.yaml

```
@@ -121,6 +121,11 @@ cephClusterSpec:
   # The default wait timeout is 10 minutes.
   waitTimeoutForHealthyOSDInMinutes: 10
 
+  # Whether or not requires PGs are clean before an OSD upgrade. If set to `true` osd upgrade process won't start until PGs are healthy.
```

First instance of OSD is capitalized. Second instance is not. We should strive for consistency of docs. I see other places where this applies also. Please update other instances too.

```suggestion
  # Whether or not requires PGs are clean before an OSD upgrade. If set to `true` OSD upgrade process won't start until PGs are healthy.
```